### PR TITLE
support converting models generated in v1.3 to 2.0 compatibility

### PR DESCRIPTION
**Pull request by njzjz**
(Next plan: add v1.2 compatibility)

### CODE REVIEW

**Comment by njzjz** [comment]
@amcadmus I've changed the implementation: Now users can converting models directly using v2.0. There's no necessary to install the old version.

File: source/op/prod_env_mat_multi_device.cc [hunk 1]
```diff
@@ -25,6 +25,26 @@ REGISTER_OP("ProdEnvMatA")
     .Output("nlist: int32");
     // only sel_a and rcut_r uesd.
 
+// an alias of ProdEnvMatA -- Compatible with v1.3
+REGISTER_OP("DescrptSeA")
```
Comment (amcadmus) [inline source/op/prod_env_mat_multi_device.cc:29]: Do we need to register `DescrptSeA`? the op name has changed to `ProdEnvMatA` by `convert`

Comment (njzjz) [reply source/op/prod_env_mat_multi_device.cc:29]: We need to load the old graph first when executing `convert`.

Comment (amcadmus) [reply source/op/prod_env_mat_multi_device.cc:29]: I see

**Review by amcadmus — APPROVED** [review]

**Review by denghuilu — APPROVED** [review]
A smart approach to register different ops with the same class ProdEnvMatA

**Pull request merged**